common/remote/sockfile.cpp

```diff
@@ -1245,7 +1275,9 @@ protected: friend class CRemoteFileIO;
         unsigned errCode;
         reply.read(errCode);
         if (errCode) {
-
+            // old Solaris dafilesrvix error code conversion
+            if ( (errCode >= 8200) && (errCode <= 8210) )
+                errCode = mapDafilesrvixCodes(errCode);
```

Comment:
A couple of lines down there is a related //TODO section, could you consider to change it as well?